key-chord.el

```diff
@@ -240,10 +250,15 @@ pressed twice.
 			   (> (prefix-numeric-value arg) 0)
 			 (not key-chord-mode)))
   (cond (key-chord-mode
-	 (setq input-method-function 'key-chord-input-method)
+	 (setq input-method-function 'key-chord-input-method
+	       key-chord-timer-object
+	       (run-with-idle-timer key-chord-safety-interval t
+				    (lambda () (setq key-chord-idle-state t))))
```

Comment:
It is better style to set just one variable per `setq` form. Stefan Monnier has said before that it was a mistake to allow setting multiple values and that the option to do this is only preserved for backward compatibility. (Same below.)

Reply:
Yes, I think so too. Fix done.